fractal_server/config.py

```diff
@@ -152,6 +152,7 @@ def DATABASE_SYNC_URL(self):
     RUNNER_ROOT_DIR: Optional[Path]
     FRACTAL_LOGGING_LEVEL: int = logging.WARNING
     FRACTAL_PUBLIC_TASK_SUBDIR: str = ".fractal"
+    SLURM_PYTHON_WORKER_INTERPRETER: Optional[str] = None
```

Comment:
Can we add a comment to describe this variable?

Reply:
done